docs/content/en/blog/news/november-2021-update.md

```diff
@@ -0,0 +1,178 @@
+    ref: v1.0.0
+    path: hello-world
+{{< /tab >}}
+{{< tab lang="yaml" header="Using Zip of source code stored in S3" >}}
+# Deploy Lambda function by a Zip packing source code stored in S3.
+apiVersion: pipecd.dev/v1beta1
+kind: LambdaFunction
+spec:
+  name: SimpleFunction
+  s3Bucket: pipecd-sample-lambda
+  s3Key: pipecd-sample-src
+  s3ObjectVersion: 1pTK9_v0Kd7I8Sk4n6abzCL
+{{< /tab >}}
+{{< tab lang="yaml" header="Using container image" >}}
+# Deploy Lambda function by using a container image.
+apiVersion: pipecd.dev/v1beta1
+kind: LambdaFunction
+spec:
+  name: SimpleFunction
+  image: ecr.ap-northeast-1.amazonaws.com/lambda-test:v0.0.1
+{{< /tab >}}
+{{< /tabpane >}}
+
+Check out its [documentation](/docs/user-guide/configuring-deployment/lambda) for more details.
+
+##### Automated Deployment Analysis (ADA)
+
+ADA is a good way to automate the process of evaluating the impact of a deployment during its lifetime to help building a robust release process. Any application can be configured to analyze its deployments based on its metrics data, log messages, or results from directly emitting requests to the application.
+
+Until now, PipeCD has been providing a way to allow the developer to configure a list `query` and `threshold` to analyze metrics data of a running deployment. Any query that returns a result exceeding the given threshold will cause the deployment to be failed, and a rollback stage will be executed if needed. Although this way is relatively straightforward, there is a challenge you can see here; it isn't easy to find an appropriate threshold value to use. And software is constantly changing time over time that making those values could be changed dynamically. Therefore, some new analysis strategies `PREVIOUS`, `CANARY_BASELINE`, `CANARY_PRIMARY` have been introduced since `v0.20.0`. They allow analyzing without requiring any threshold values.
+
+Sum up, you can choose one of the following four strategies which fits your use case:
+
+- `PREVIOUS`: A method to compare the time series of metrics data between the running and the last successful deployment
+- `CANARY_BASELINE`: A method to compare the time series of metrics data between the Canary and Baseline variants
+- `CANARY_PRIMARY`: A method to compare the time series of metrics data between the Canary and Primary variants
+- `THRESHOLD`: A method to check the given queries againts the given thresholds
+
+{{< tabpane >}}
+{{< tab lang="yaml" header="Using PREVIOUS strategy" >}}
+# ADA by comparing between metrics data of current and previous deployment.
+- name: ANALYSIS
+  with:
+    duration: 30m
+    metrics:
+    - strategy: PREVIOUS
+      provider: my-prometheus
+      deviation: HIGH
+      interval: 5m
+      query: sum (rate(http_requests_total{status=~"5.*"}[5m])) / sum (rate(http_requests_total[5m]))
+{{< /tab >}}
+{{< tab lang="yaml" header="Using CANARY_BASELINE strategy" >}}
+# Deploy Lambda function by a Zip packing source code stored in S3.
```

Comment:
This note is not match the sample snipet

Reply:
Nice catch!
Fixed it!